language/moe/consolidate_results.py

```diff
@@ -0,0 +1,116 @@
+import argparse
```

Comment:
what is this script for? why is it only for open orca?

Reply:
Not sure. For now it is removed. I will inspect it to check if we need to add it again